code/modules/antagonists/eldritch_cult/knowledge/ash_lore.dm

```diff
@@ -41,10 +41,10 @@
 				continue
 			var/obj/effect/proc_holder/spell/targeted/touch/mansus_grasp/MG = X
 			MG.charge_counter = min(round(MG.charge_counter + MG.charge_max * 0.75),MG.charge_max) // refunds 75% of charge.
-	var/atom/throw_target = get_edge_target_turf(C, user.dir)
-	if(!C.anchored)
-		. = TRUE
-		C.throw_at(throw_target, rand(4,8), 14, user)
+	to_chat(C, "<span class='danger'>Your eyes burn horrifically!</span>") //pocket sand! also, this is the message that changeling blind stings use, and no, I'm not ashamed about reusing it
+	C.become_nearsighted(EYE_DAMAGE)
+	C.blind_eyes(10)
+	C.blur_eyes(20)
```

Comment:
are these times?

Needs time defines if so.

Reply:
these values are expressed in seconds already

using "SECONDS" would be misleading here, as I'd have to use something like "20 SECONDS" for an effect that lasts for only 2 seconds.